src/runtime/composables/useAppManager.ts

```
@@ -103,10 +103,22 @@ export const useAppManager = defineStore("app-manager", {
             if (error){ console.log('حدث خطأ اثناء شراء الخدمة'); return false }
             if (!data) { console.log('لاتمتلك مايكفي من النقاط'); return false }
             this.fetch();
-        }
+        },
+
+        // Buy an package
+        async buyPack(pack_id){
+            const supabase = useSupabaseClient()
+
+            let { data, error } = await supabase.functions.invoke('core-buy-pack', {
+                    body: JSON.stringify({ pack_id: pack_id }),
+                })
+            if (error)console.log('حدث خطأ اثناء شراء الباقة')
```

This should not write the error in the console. Because it will not be useful.
When call `buyPack(pack_id)` it should return the specific error in return.
For example:
```ts
if (error) { 
   switch(error){
       case 'first error':
           return 'There is problem. First error.';
       case default:
           return 'There is unknown error.
   }
}
```
Or even it could be:
```ts
if (error) return error
```

Thank you

---

I completed returning the specific error in `return`.
 You can check it.